Sources/Sentry/Public/SentryHub.h

```diff
@@ -196,13 +202,15 @@ SENTRY_NO_INIT
 
 /**
  * Checks if a specific Integration (`integrationClass`) has been installed.
- * @return BOOL If instance of `integrationClass` exists within
- * `SentryHub.installedIntegrations`.
+ *
+ * @return BOOL If instance of `integrationClass` exists within `SentryHub.installedIntegrations`.
  */
 - (BOOL)isIntegrationInstalled:(Class)integrationClass;
 
 /**
- * Set global user -> thus will be sent with every event
+ * Set user to the Scope of the Hub. Thus, the Hub is going to send them with every event.
```

Comment:
```suggestion
 * Set user to the Scope of the Hub.
```

this is pretty much the same for all the Scope fields, eg addBreadcrumb, and we don't say that, I'd just skip it as its the intended behaviour anyway